REVIEWING.md

```diff
@@ -0,0 +1,122 @@
+If this is not possible, deferring the PR to the next breaking change window or closing it entirely is appropriate.
+
+### Commit Messages and Hygiene
+
+Good commit messages are essential for understanding why a change was made in the future.
+Reviewers should check each commit of a PR to ensure that it has an appropriate commit message which fully and accurately explains the change and why it is being made.
+This should remain true after changes to the PR due to code review, so please re-review commit messages before merging code, to ensure they are still accurate.
+Pull requests fixing a specific issue must include a `Fixes: #xxxxx` line in the commit message.
+Full details on the `Fixes` line can be found in the [Contributor's Guide](./CONTRIBUTING.md).
+Reviewers are responsible for enforcing the guidelines in that document.
+
+Each commit in a PR should be self-contained and have a clear and distinct purpose.
+If this is not true, please encourage the contributor to squash their commits using `git rebase -i` until it is true.
+
+### Disagreements between reviewers
+
+We do not expect all reviewers to be of the same opinion during code review.
+If you see another reviewer requesting changes to a PR you do not agree with, it is perfectly acceptable to comment to that effect.
+Disagreements between maintainers can generally be worked out during PR review through comments.
+If this is not possible, other maintainers can be called on to give their opinions and determine a way forward.
+We encourage such disagreements and discussions, so long as they remain respectful and are done with the goal of resolving the dispute.
+
+If a decision cannot be reached, the issue may be put to a vote, in which all Maintainers of the repository in question and all Core Maintainers can vote.
+
+Being respectful includes respect for the contributor's time.
+If there is a disagreement, please do not make the contributor make repeated changes until an agreement on how to proceed is reached.
+Also, please attempt to reach an agreement quickly, so the PR can be merged in a timely fashion.
+
+### Language Version Updates
+
+Most repositories in the Podman project are written in Go, and as such target a specific version of Go in their `go.mod`.
+For example:
+```
+$ cat go.mod | grep 'go 1.'
+go 1.22.8
+```
+
+Changing this value affects what versions of Go can build the project, and as such increasing it can prevent some distributions with older Golang versions from building Podman.
+
+For all branches except the main branch, Go version should remain static unless there is an extremely good reason to change it (for example, a CVE fix requires pulling in a new version of a library that needs a newer Golang version).
+PRs into a non-main branch which change the supported Go version should be modified to not require such a change, or rejected if that is not possible.
+Changing supported Go version in the main branch is allowed, but not encouraged.
+
+### Tests
```

Comment:
Documentation changes and certain changes don't require `tests` for such cases `No New test` label is used, should it reflect here ?